packages/forms/src/styled/input-group/StyledInputGroup.ts

```diff
@@ -79,6 +80,10 @@ const itemStyles = (props: ThemeProps<DefaultTheme>) => {
       z-index: -1;
     }
 
+    & > ${StyledTextInput}:first-child {
+      margin-${horizontal}: 0 !important; /* [2] */
```

Comment:
Would moving this to the end of the style cascade make `!important` unnecessary? 🤔

Reply:
Just updated changes to remove `!important` altogether.